region/maridia/inner-pink/East Cactus Alley Room.json

```diff
@@ -1136,10 +1133,6 @@
             "canJumpIntoIBJ",
             "h_canUseSpringBall"
           ]},
-          {"and": [
```

Comment:
I felt like the bomb jump water escape was so much harder than turning off hijump to jump across. But I can add it back if you think it could be useful?

Reply:
I think the bomb jump is harder and the spike jump is less apparent.  So I see them as similarly difficult but it would be more useful if the logic told people to spike jump.